public/controllers/management/components/management/configuration/office365/constants.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Wazuh app - Constants of Office365 configuration
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+export const OFFICE_365 = 'office365';
+export const WMODULES_WMODULES = 'wmodules-wmodules';
+export const HELP_LINKS = [
+  {
+    text: 'Using Wazuh to monitor Office 365',
+    href: 'https://documentation.wazuh.com/current/office365/index.html',
+  },
+  {
+    text: 'Configuration options of the Office 365 module reference',
```

Comment:
`Configuration options for the Office 365 module`
I think `for` is better, maybe we should change in the other modules too